conventions/documentation/actionmenus.md

```diff
@@ -91,3 +91,18 @@ Actions in the menu should follow the following pattern:
 - Delete [Resource Name(s)]
 
 ---
+
+## Hiding and disabling actions
```

Comment:
Love seeing this convention addition! I guess I was wondering if we even reframed the 5 bullet points below as in fact the conventions (beyond being guidelines or recommendations,) with some caveat that they can be bent if some exception is needed, would that be too heavy handed? They way they are written to me seems like those 5 bullets could be the rules to follow in the console.

Reply:
Yeah that's a fair point. I purposely made it a little wishy washy in the draft since we weren't sure what the guidelines would end up looking like, but I agree, we could just say these are the conventions 👍